### DISCUSSION TITLE
关于 “在线版本将转为维护模式”

### DISCUSSION THREAD

**Discussion by linzeliang1222** [SPlayer-Dev/SPlayer#166] [Q&A] [answered]
“在线版本将转为维护模式” 请问这个是指网页版本后续不更新了还是什么意思呢，如果自己build项目到服务器部署是最新的吗

**Comment by imsyy** [comment, marked as answer]
不会添加额外功能，仅在现有功能上进行维护

**Reply by linzeliang1222** [reply]
那请问使用 Docker 部署的是最新的功能吗

**Reply by imsyy** [reply]
> 那请问使用 Docker 部署的是最新的功能吗

目前的网页版、客户端以及Docker版本均为“在线版”，意为需要使用网络以及第三方api进行使用

**Reply by linzeliang1222** [reply]
感谢解答，后续开发方向本地播放器也是在当前版本继续开发的吧，那 Docker 部署的网页版还能使用到之后新功能或改进吗
